build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+plugins {
+    `kotlin-dsl`
+    `java-gradle-plugin`
+    id("maven-publish")
+    id("com.gradle.plugin-publish").version("0.16.0")
+}
+
+repositories {
+    mavenCentral()
+}
+
+group = "com.doist.gradle"
+version = property("version") as String
+
+dependencies {
+    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin")
+}
+
+val pluginName = "Ktlint"
+
+gradlePlugin {
+    plugins.register(pluginName) {
+        id = "${project.group}.${project.name}"
+        implementationClass = "com.doist.gradle.KtlintPlugin"
+    }
+    isAutomatedPublishing = true
+}
+
+pluginBundle {
+    website = "https://github.com/Doist/ktlint-gradle-plugin"
+    vcsUrl = "https://github.com/Doist/ktlint-gradle-plugin.git"
+
+    plugins.getByName(pluginName) {
+        displayName = "Ktlint Gradle Plugin"
+        description = "This plugin is mainly designed to check code styles on changed part of the code"
```

Comment:
Suggestion: _"This plugin is designed to validate the code style of changed kt files."_